lib/athena_health/endpoints/configurations.rb

```diff
@@ -21,6 +21,17 @@ def all_medications(practice_id:, search_value:)
         response.map { |medication| AthenaHealth::Medication.new(medication) }
       end
 
+      def all_procedures(practice_id:, search_value:)
+        response = @api.call(
+          endpoint: "#{practice_id}/reference/order/procedure",
+          method: :get,
+          params: { searchvalue: search_value }
```

Comment:
Nit: What search value would we use for getting all procedures? Or is this method really get_procedures where no search_value returns all and a specified value will return a subset?

Reply:
Good point, I named it all_ to fit the existing convention this gem has when searching for allergies and medications. Agreed it's misleading since it can never return the entire list

Reply:
I agree that it's slightly odd that these methods are prefixed by `all_`, but it probably makes sense to follow the existing convention.